packages/timeverter/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=timeverter
+pkgver=83.24b715e
+pkgrel=1
+pkgdesc='Bruteforce time-based tokens and convert several time domains.'
+groups=('blackarch' 'blackarch-cracker')
+arch=('any')
+url='https://github.com/D3vil0p3r/timeverter'
+license=('GPL3')
+depends=('python')
+makedepends=('git')
+source=("git+https://github.com/D3vil0p3r/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $pkgname
+  mkdir -p "$pkgdir/usr/share/timeverter/src"
+  mkdir -p "$pkgdir/usr/bin"
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/" timeverter.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/core.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/parser.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/printer.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/time.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/util.py
+  ln -s "$pkgdir/usr/share/timeverter/timeverter.py" "$pkgdir/usr/bin/timeverter"
```

Comment:
`from src.core import main` won't work with this deployment. So you'll probably need a jointscript. cf. https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-jointscript

Reply:
Thank you for the answer @noraj I updated it on the commit cbe35d1c40166ba84b3f4bd8c1299f825a77db9a